packages/main/src/Carousel.js

```diff
@@ -95,24 +95,12 @@ const metadata = {
 		},
 
 		/**
-		 * Defines the index of the initially selected item.
-		 * @type {Integer}
-		 * @defaultvalue 0
-		 * @public
-		 */
-		selectedIndex: {
-			type: Integer,
-			defaultValue: 0,
-		},
-
-		/**
-		 * Defines when the <code>load-more</code> event is fired. If not applied the event will not be fired.
+		 * Defines the initially selected page.
```

Comment:
```selectedIndex``` should remain as private property. This should be added as BREAKING_CHANGE in the description of the PR